sources/ham/tests/ham-test-thrift/_build.ham

```diff
@@ -1,12 +1,15 @@
-SubDir TOP ;
+MODULE_NAME = ham-test-thrift ;
```

Comment:
I think we shouldn't do MODULE_NAME anymore. I know I did this in some RDD repos - and maybe some other places - but its actually harder to maintain than just inlining it.

Mostly it encourages you to make contrieved combinations like $(MODULE_NAME)_some_suffix which just become quickly an unreadable mess.

Its really not hard to s/ham-test-thrift/ham-test-my-new-name/g if you rename it and its a lot more readable.